### PR TITLE
Initial version of annotation_surveyor

### PR DESCRIPTION
**Pull request by bwilkerson**

### CODE REVIEW

File: example/annotation_surveyor.dart [hunk 1]
```diff
@@ -0,0 +1,198 @@
+//  Copyright 2019 Google LLC
```
Comment (pq) [inline example/annotation_surveyor.dart:1]: > 2020?

**Comment by pq** [comment]
Also, did your run `dartfmt`?

https://travis-ci.org/github/pq/surveyor/jobs/711171754

File: example/annotation_surveyor.dart (hunk 1 above)
Comment (bwilkerson) [reply example/annotation_surveyor.dart]: Done

**Comment by bwilkerson** [comment]
Formatted

**Comment by pq** [comment]
Awesome.  

Fix for the  warning in `driver` here: https://github.com/pq/surveyor/pull/23

**Pull request merged**